extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1087,7 +1101,8 @@ export class AmpA4A extends AMP.BaseElement {
     dev().assert(!!this.element.ownerDocument, 'missing owner document?!');
     this.protectedEmitLifecycleEvent_('renderFriendlyStart');
     // Create and setup friendly iframe.
-    const iframe = /** @type {!HTMLIFrameElement} */(
+    dev().assert(!this.iframe);
```

Comment:
You can remove these now given we have an explicit check earlier in layoutCallback